src/store/reducers/exampleReducer.js

```diff
@@ -0,0 +1,15 @@
+const initialState = { counter: 0 };
+
+const exampleReducer = (state = initialState, action) => {
+    switch(action.type){
+        case 'ADD_ACTION':
```

Comment:
I remember that we had an agreement that you will create new action (in example multiplyer), which will be able to receive argument and use it.